README.md

```diff
@@ -106,6 +106,14 @@ cd fasp-documentation
 npm install
 ```
 
+#### Local Development Setup
```

Comment:
If I was being picky, I would suggest this should have gone into a separate commit, but it's no biggie to have included it in this one.